recipes/gdal2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: gdal
+  version: 2.1.0.trunk
+  
+source:
+  svn_url: https://svn.osgeo.org/gdal/trunk/gdal
```

Comment:
Yes, latest layer intersection function implemented from geos with some special tests to speed up processing (by scales). This is no showstopper, gdal maintainers are about to release version 2.1 in april/may this year anyway.